basics/a106.go

```diff
@@ -0,0 +1,16 @@
+// Simple Input program
+
+package main
+
+import (
+	"fmt"
+)
+func main() {
+	var answer string
+	fmt.Println("Is zero a whole number (Y/N):")
+	_, err := fmt.Scan(&answer)
+	if err != nil {
```

Comment:
With fmt.Scan, err will never be nil. Or can it be?

Reply:
err is nil by default if scan reads the input from stdin properly and it can save the arguments passed to variables successfull. 


```
package main
import "fmt"

func main() {
var answer string
fmt.Println("Is zero a whole number (Y/N):")
_, err := fmt.Scan(&answer)
fmt.Println("error = ", err)
}
```

```
[mniranja@mniranja slices]$ ./a1 
Is zero a whole number (Y/N):
y
error =  <nil>
```